Limelight/Stream/VideoDecoderRenderer.m

```diff
@@ -106,38 +118,37 @@ - (void)start
         _displayLink.preferredFramesPerSecond = self->frameRate;
     }
     [_displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:NSDefaultRunLoopMode];
+
+}
+
+- (void) setupDecompressionSession {
+    if (decompressionSession != NULL){
+        VTDecompressionSessionInvalidate(decompressionSession);
+        CFRelease(decompressionSession);
+        decompressionSession = nil;
+    }
+    
+    int status = VTDecompressionSessionCreate(kCFAllocatorDefault,
```

Comment:
For initialize VTDecompressionSession need set parameters.
If I understand it correctly, the sender can send data from various sources.
Therefore, it is necessary to prepare for the fact that the data can be of various types.
For example:
```
        let imageBufferAttributes = [
            //kCVPixelBufferPixelFormatTypeKey: NSNumber(value: kCVPixelFormatType_......), /// if needed
            kCVPixelBufferIOSurfacePropertiesKey: [:] as AnyObject, 
            kCVPixelBufferOpenGLESCompatibilityKey: NSNumber(booleanLiteral: true), 
            kCVPixelBufferMetalCompatibilityKey: NSNumber(booleanLiteral: true), 
            kCVPixelBufferOpenGLCompatibilityKey: NSNumber(booleanLiteral: true) 
        ]
```